code/game/machinery/doors/airlock.dm

```diff
@@ -1072,10 +1072,33 @@
 		user.visible_message("<span class='notice'>[user] pins [C] to [src].</span>", "<span class='notice'>You pin [C] to [src].</span>")
 		note = C
 		update_icon()
+	else if(HAS_TRAIT(C, TRAIT_DOOR_PRYER) && user.a_intent != INTENT_HARM)
+		if(isElectrified())
+			shock(user,100)
```

Comment:
This should take into account siemens coefficient of the item

Reply:
👍